src/private/teams.ts

```diff
@@ -51,13 +51,17 @@ export namespace teams {
    * @param threadId ID of the thread where the app entity will be created; if threadId is not
    * provided, the threadId from route params will be used.
    */
-  export function refreshSiteUrl(callback: (error: SdkError) => void, threadId?: string): void {
+  export function refreshSiteUrl(threadId: string, callback: (threadId: string, error: SdkError) => void): void {
     ensureInitialized();
 
+    if (!threadId) {
+      throw new Error('[teams.refreshSiteUrl] threadId cannot be null or empty');
+    }
+
     if (!callback) {
       throw new Error('[teams.refreshSiteUrl] Callback cannot be null');
     }
 
-    sendMessageToParent(threadId);
+    sendMessageToParent('teams.refreshSiteUrl', [threadId], callback);
```

Comment:
There shouldn't be any actual code changes in the release branch. Please check those into master separately.

Reply:
This line of code is in master code base :)